### DISCUSSION TITLE
Getting error on artillery github actions pipeline for playwright tests

### DISCUSSION THREAD

**Discussion by dushyant1307** [artilleryio/artillery#2448] [Artillery]
I am pretty new to github actions and artillery can someone please help me in understanding why On github actions error is thrown `worker error, id: 1 browserType.launch: Executable doesn't exist at /github/home/.cache/ms-playwright/chromium-1084/chrome-linux/chrome` but the binaries are installed in `Chromium 119.0.6045.9 (playwright build v1084) downloaded to /home/runner/.cache/ms-playwright/chromium-1084`

Pipeline looks like below:
name: Load Test

on:
    workflow_dispatch:
 
jobs:
  artillery:
    runs-on: ubuntu-latest
 
    steps:
      - name: Checkout code
        uses: actions/checkout@v3

      - name: Set up Node.js
        uses: actions/setup-node@v3
        with:
          node-version: 18

      - name: Install dependencies
        run: |
          npm install
          npx playwright@1.39.0 install
          
      - name: Execute load tests
        uses: artilleryio/action-cli@v1
        with:
          command: run hello-world.yml --output ./report.json
        
      - name: Upload artifact
        uses: actions/upload-artifact@v3
        if: always()
        with:
          name: artillery-report
          # Reference the generated report in the file system.
          path: ./report.json

**Comment by hassy** [comment]
This may be due to the mismatch between Playwright versions. The latest version of Artillery uses Playwright 1.41.0 (https://github.com/artilleryio/artillery/blob/main/packages/artillery-engine-playwright/package.json#L15) but you're installing Playwright 1.39.0
